.gitattributes

```diff
@@ -1,3 +1,6 @@
 test/fixtures/scripts/needs-compile.js eol=crlf
 test/fixtures/scripts/needs-compile.compiled.js eol=crlf
-test/fixtures/scripts/needs-compile.compiled.js.map eol=crlf
\ No newline at end of file
+test/fixtures/scripts/needs-compile.compiled.js.map eol=crlf
+test/fixtures/scripts/branches.covered.js eol=lf
```

Comment:
The tests were originally failing in windows. That was because of git replacing LF with CR/LF